python/llm/scripts/ipex-llm-init-support.py

```diff
@@ -0,0 +1,122 @@
+import os
+import sys
```

Comment:
This python scripts may also need to be put into here: https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/setup.py#L337 to make sure users can directly use it through ipex-llm-init after installing ipex-llm